<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SharonS" failures="2" tests="9" name="TestRE" time="0.202" errors="0" timestamp="2023-09-14T11:20:29 IST">
  <testcase classname="org.pageexe.UnKnownFails" name="test" time="0.027">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.pageexe.UnKnownFails" name="test" time="0.007">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.pageexe.UnKnownFails" name="test" time="0.008">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.pageexe.UnKnownFails" name="test" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.pageexe.UnKnownFails.test(UnKnownFails.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="org.pageexe.UnKnownFails" name="test1" time="0.003">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="org.pageexe.UnKnownFails" name="test1" time="0.01">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="org.pageexe.UnKnownFails" name="test1" time="0.003">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="org.pageexe.UnKnownFails" name="test1" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.pageexe.UnKnownFails.test1(UnKnownFails.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="org.pageexe.UnKnownFails" name="test2" time="0.007"/>
</testsuite> <!-- TestRE -->
